frontend-react/playwright.config.ts

```diff
@@ -81,35 +81,29 @@ export default defineConfig<TestOptions>({
     },
 
     projects: [
-        /* Test setup (ex: authenticated sessions) */
         { name: "setup", testMatch: /\w+\.setup\.ts$/ },
+        // We have a suite of tests that are ONLY checking links so to
+        // save bandwidth, we only need to utilize a single browser
+        {
+            name: "chromium-specific",
+            use: { browserName: "chromium" },
+            testMatch: "spec/chromium-only/*.spec.ts",
```

Comment:
Are there expectations of checking links on pages that require permissions? If so, you will need to include the setup project in the dependencies property and create tests that use the authenticated json context files.

Reply:
Nope, this test is EXPLICITLY for public, non-authenticated pages only.